plan/physical_plans.go

```diff
@@ -156,9 +156,9 @@ type PhysicalIndexScan struct {
 	DoubleRead bool
 
 	// accessInAndEqCount is counter of all conditions in AccessCondition[accessEqualCount:accessInAndEqCount].
```

Comment:
update comment

Reply:
Thank you for the sugguestion. I've changed the comments as follows:

 `-  // accessInAndEqCount is counter of all conditions in AccessCondition[accessEqualCount:accessInAndEqCount].`
 `+  // AccessInAndEqCount is counter of all conditions in AccessCondition[AccessEqualCount:AccessInAndEqCount].`
 `   AccessInAndEqCount int`